### ISSUE TITLE
"unable to get local issuer certificate" when using internal CA

### ISSUE THREAD

**Issue by jpcoder2** [jabbera/vsts-git-release-tag#41]
We are getting the error "unable to get local issuer certificate" when trying to create a branch in a release definition.  I'm guessing this is because we are using an internal CA for the server's SSL cert.  We had similar issues when setting up git with builds but go around it by adding a crt file containing the internal CAs to the .gitconfig file.  Will this not work for this extension?  If so, is there another way to get it to work?

Thanks...

**Comment by jabbera** [comment]
Hi,

Sorry for the delay. Did you sort this? I don't have any experience with this, but MSFT makes a ton of recommendations here: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/certificate?view=tfs-2018

**Comment by jpcoder2** [comment]
No, I haven’t figured this out but haven’t looked at it since I posted this either. Since TFS build and release works fine with Git and our certs I’m guessing this is something odd your extension is doing but I don’t know what that would be.

**Comment by jabbera** [comment]
I just use the vsts api directly. This is a node based task and node typically uses it's own cert store and not the windows store. You might try setting this environment variable:

https://nodejs.org/api/cli.html#cli_node_extra_ca_certs_file

**Comment by theDeployinator** [comment]
I was having the same error. Setting the NODE_EXTRA_CA_CERTS environment variable on the TFS agent to point to a file containing our internal self-signed root certificate resolved the "unable to get local issuer certificate" error for me. The git tags now work as expected.

**Comment by jabbera** [comment]
Thanks for the update!
